src/modules/Blur/Module.js

```diff
@@ -15,7 +15,7 @@ module.exports = function Blur(options, UI) {
         var step = this;
 
         function extraManipulation(pixels) {
-            pixels = require('./Blur')(pixels, options.blur);
+            pixels = require('./Blur')(pixels, parseFloat(options.blur));
```

Comment:
@HarshKhandeparkar I think parseFloat can be done seperately so that its clear.

Reply:
Do you mean like `options.blur = parseFloat(options.blur)`?